src/main/java/uk/gov/beis/els/api/categories/refrigeratordirectsales/BeverageCoolersInternetLabelApiForm.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.beis.els.api.categories.refrigeratordirectsales;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import javax.validation.constraints.NotBlank;
+import uk.gov.beis.els.api.common.ApiValuesFromLegislationCategory;
+import uk.gov.beis.els.api.common.RescaledInternetLabelApiForm;
+import uk.gov.beis.els.categories.refrigeratorsdirectsales.service.RefrigeratorsDirectSalesService;
+
+@Schema(name = "Beverage coolers internet label")
```

Comment:
all internet label API forms are the same, but I couldn't come with a way of making the schema name different for each category.
Happy to reworked if there is a solution to that

Reply:
We could just have 1 internet label form for all these categories, and call the schema something like `Arrow image for refrigerators with a direct sales function`. The single field could just be called `Energy efficiency class indicator` to keep it generic, which is what the current HTML forms do for this category.

Not sure if @matteason has an opinion on which is better?

Reply:
I agree with the redundancy, will wait on @matteason opinion to make changes